code/modules/cargo/packs/livestock.dm

```diff
@@ -133,7 +133,7 @@
 	desc = "The fox goes...? Contains one fox. Comes with a collar!"//what does the fox say
 	cost = CARGO_CRATE_VALUE * 10
 	access_view = ACCESS_CAPTAIN
-	contains = list(/mob/living/simple_animal/pet/fox,
+	contains = list(/mob/living/basic/pet/fox,
 					/obj/item/clothing/neck/petcollar,
 				)
```

Comment:
```suggestion
	contains = list(
		/mob/living/basic/pet/fox,
		/obj/item/clothing/neck/petcollar,
	)
```

may as well fix this ugly looking list since i don't think anyone else will ever touch this ever again